### ISSUE TITLE
Show a list widget if ili2db.mapping = ARRAY

### ISSUE THREAD

**Issue by m-kuhn** [opengisch/QgisModelBaker#597]
If a JSON field from a GeoPackage DB has a ili2db.mapping=ARRAY meta-attribute it should be loaded with the List edit widget rather than with a Key/Value one.

**Comment by signedav** [comment]
To create it:
```
INTERLIS 2.3;

MODEL JSON_Array (en) 
AT "https://signedav.github.io/usabilitydave/models"
VERSION "2020-06-22" =
    TOPIC JSON_Arr =
        STRUCTURE Item = 
            Name: TEXT;
        END Item;
        CLASS TheClass =
            !!@ili2db.mapping=ARRAY
            ItemsArray: LIST {0..*} OF Item;
        END TheClass;
        
    END JSON_Arr; !! of TOPIC

END JSON_Array. !! of MODEL

```

**Comment by signedav** [comment]
Solved here #607
